app/src/main/java/org/literacyapp/startguide/content/swipe/SwipeUpDownActivity.java

```diff
@@ -5,31 +5,134 @@
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import android.view.View;
 
 import org.literacyapp.startguide.R;
 import org.literacyapp.startguide.util.AnimationHelper;
+import org.literacyapp.startguide.util.MediaPlayerHelper;
 
 
 /**
  *
  */
-public class SwipeUpDownActivity extends AppCompatActivity {
+public class SwipeUpDownActivity extends AppCompatActivity implements GestureDetector.OnGestureListener {
+
+    private GestureDetector mGestureDetector;
+    private View mHandView;
+    private boolean detectUp = true;
+    private boolean detectDown = false;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.activity_swipe_up_down);
 
+        //List of images
         TypedArray imagesArray = getResources().obtainTypedArray(R.array.image_files);
         SwipeUpDownAdapter adapter = new SwipeUpDownAdapter(imagesArray);
 
         RecyclerView recyclerView = (RecyclerView) findViewById(R.id.recyclerView);
         recyclerView.setLayoutManager(new LinearLayoutManager(this));
         recyclerView.setAdapter(adapter);
 
-        // TODO: 04/02/2017 play "Swipe to scroll up"
-        //MediaPlayerHelper.play(this, R.raw.);
+        //Hand view
+        mHandView = findViewById(R.id.hand);
+
+        //Add gesture detector
+        mGestureDetector = new GestureDetector(this, this);
+        recyclerView.setOnTouchListener(new View.OnTouchListener() {
+            @Override
+            public boolean onTouch(View v, MotionEvent event) {
+                return mGestureDetector.onTouchEvent(event);
+            }
+        });
+
+        showScrollUp();
+    }
+
+    /**
+     * Scroll up explanation with audio and hand animation
+     */
+    private void showScrollUp() {
+        //TODO: 04/02/2017 update audio file (en, sw) "Swipe to scroll up"
+        MediaPlayerHelper.play(this, R.raw.scroll_up);

```

Comment:
I was confused when the task was to scroll down to the bottom of the list and the voice said "Swipe to scroll up". Can we replace the audio instruction with something not as technical as "swipe" and "scroll"? If the children have never used a tablet, it's likely that "swipe" and "scroll" are unfamiliar terms.

How about something like this?: "Move to the bottom of the list?" or "Can you move to the bottom if the list?" or "Can you find the animals/images at the bottom of the list?"